.github/workflows/dev-deploy-cloud-functions.yml

```
@@ -0,0 +1,88 @@
+      - '*FIT-**'
+    paths:
+      - 'cloud_functions/**'
+      - '.github/workflows/*-deploy-cloud-functions.yml'
+
+env:
+  PROJECT_ID: ${{ secrets.PROJECT_ID }}
+  REGION: us-west1
+  ENVIRONMENT: dev
+  ENCRYPTION_SECRET: ${{ secrets.ENCRYPTION_SECRET }}
+jobs:
+#  test:
```

YAMLlint (1.35.1)

[warning] comment not indented like content

(comments-indentation)